ui-guidelines/css/resources.css

```diff
@@ -0,0 +1,283 @@
+/*==== Resources TYPOGRAPHY ====*/
+.typography-weights {
+	width: 100%;
+	height: 10em;
+}
+.weight-sample {
+	width: 20%;
+	float: left;
+	margin-right: 5%;
+	border: 1px solid #fce3da;
+	border-radius: 8px;
+}
+.big-letter{
+	font-size: 6em;
+	font-family: 'Open Sans', Arial, sans-serif;
+}
+.light {
+    font-weight: 300;
+}
+.normal {
+    font-weight: 500;
+}
+.medium {
+    font-weight: 600;
+}
+.bold {
+    font-weight: 700;
+}
+.text-size {
+    margin-left: 1.5em;
+}
+.typography {
+	font-family: 'Open Sans', Arial, sans-serif;
+}
+.typography p {
+    font-family: inherit;
+}
+h1 {
+    font-size: 2.5rem;
+}
+h2 {
+    font-size: 2rem;
+}
+h3 {
+    font-size: 1.75rem;
+}
+h4 {
+    font-size: 1.5rem;
+}
+h5 {
+    font-size: 1.25rem;
+}
+h6 {
+    font-size: 1rem;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-weight: bold;
+}
+
+.text-x-huge {
+    font-size: 2.5rem;
+}
+.text-huge {
+    font-size: 1.875rem;
+}
+.text-xx-large {
+    font-size: 1.4rem;
+}
+.text-x-large {
+    font-size: 1.133rem;
+}
+.text-large {
+    font-size: 1.1rem;
+}
+.text-xx-medium {
+    font-size: 1.066rem;
+}
+.text-x-medium {
+    font-size: 1rem;
+}
+.text-medium {
+    font-size: .933rem;
+}
+.text-small {
+    font-size: .9rem;
+}
+.text-x-small {
+    font-size: .866rem;
+}
+.text-xx-small {
+    font-size: .85rem;
+}
+.text-minimum {
+    font-size: .8rem;
+}
+.text-x-minimum {
+    font-size: .75rem;
+}
+.text-xx-minimum {
+    font-size: .625rem;
+}
+
+/*==== COLORS ====*/
+hr {
+	margin-top: 0px;
+	border-top: 1px solid #333;
+}
+.color-row {
+	display: flex;
+    justify-content: flex-start;
```

Comment:
CS: Inconsistent indenatation. One time it's 8 spaces , another time it's 4 spaces. I think it is a good idea to choose one and be consistent, do you agree with me?